test/fixtures/minio-fixture/src/main/java/org/elasticsearch/test/fixtures/minio/MinioTestContainer.java

```diff
@@ -15,9 +15,10 @@
 
 public final class MinioTestContainer extends DockerEnvironmentAwareTestContainer {
 
-    // NB releases earlier than 2025-05-24 are buggy, see https://github.com/minio/minio/issues/21189, and #127166 for a workaround
-    // However the 2025-05-24 release is also buggy, see https://github.com/minio/minio/issues/21377, and this has no workaround
-    public static final String DOCKER_BASE_IMAGE = "minio/minio:RELEASE.2025-06-13T11-33-47Z";
+    // NB releases earlier than 2025-05-24 are buggy, see https://github.com/minio/minio/issues/21189, and #127166 for a workaround.
```

Comment:
Is https://github.com/minio/minio/issues/21189 mentioned because the bug wasn't fixed properly, leading to you filing [21456](https://github.com/minio/minio/issues/21456)?

I'm wondering if it's worth mentioning resolved bugs. But if it still relates, then that seems helpful, maybe mentioned in that way.

Reply:
I'm not sure, I suspect https://github.com/minio/minio/issues/21456 might be something different albeit with similar symptoms.